Dockerfile.image-service-build

```diff
@@ -8,13 +8,17 @@ RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/i
 
 FROM quay.io/centos/centos:stream9
 
+RUN dnf install -y epel-release
```

Comment:
nit: Using one RUN statement instead of two would generate one layer instead of two